XSUP-50225 OracleCloudInfrastructureEventCollector (#39846)
demisto/content: Packs/OracleCloudInfrastructure/ReleaseNotes/1_0_29.md

#### Integrations

##### Oracle Cloud Infrastructure Event Collector

> Fixed an issue where the private key parameter was not parsed correctly.

Reviewer: ```suggestion
- Fixed an issue where the private key parameter was not parsed correctly.
```

> - Fixed an issue where the private key validation function incorrectly handled keys with inline content.

Reviewer: ```suggestion
Fixed an issue where the private key was not parsed correctly.
```
